assets/src/components/SelectCard.js

```diff
@@ -159,7 +159,7 @@ const SelectCard = props => {
     const label = 'About ' + cardData.title
     if (cardData.helpUrl) {
       return (
-        <div class={classes.help}>
+        <div className={classes.help}>
```

Comment:
Oddly enough, even though there was a warning about an invalid DOM property, the classes were still being applied. Otherwise I don't think my solution from #1475 would have worked.